scripts/ci/build_push_images.sh

```diff
@@ -1,51 +0,0 @@
-#!/bin/bash -xe
-
-# Validations
-MANDATORY_ENVS="IMAGE_VERSION BUILD_NUMBER DOCKER_REGISTRY CSI_NODE_IMAGE CSI_CONTROLLER_IMAGE GIT_BRANCH"
-for envi in $MANDATORY_ENVS; do 
-    [ -z "${!envi}" ] && { echo "Error - Env $envi is mandatory for the script."; exit 1; } || :
```

Comment:
where did this check go?

Reply:
done

Reply:
this "done" raises a lot of questions, to name a few:

- where did `#!/bin/bash -xe` go?
- where did the newline at the EOF go?
- how can one understand the context of this file/variables from the `run_variables_verification.sh` name?

Reply:
to which name do you suggest that I will change it?

Reply:
I assume `verify_env_vars_for_build_push.sh` would add more context, but this is just 1 of the many bad symptoms of the unnecessary separation of our code.

Reply:
removed it, done